src/css/var/autoPx.js

```diff
@@ -0,0 +1,63 @@
+define( function() {
+
+"use strict";
+
+return {
```

Comment:
As there's a lot of repetition here, would it be possible to build this using a combination of a few keywords?  

Granted this file should probably be static, but: a handful of these are permutations of ["margin","border","padding"] + [...direction...] + ["", "width", "height"] and some form of population function for those combinations could happen?

Reply:
Additionally, as the constant "true" is used in all these cases-- could the number `1` be used instead to reduce space?

Reply:
Between Uglify and the zgip compression, many of these things go away. We don't generally care about the unminified size so things like `true` to `1` don't help. It's hard to know for sure if it's smaller after min+gzip unless you do the permutations and see what happens.

Reply:
@gvanderest In addition to what @dmethvin said, in JS there's no way to quickly compute a list of all props with a certain prefix, infix & suffix; we'd need to add a triple for-loop and that adds size, not reduces it.

Instead of constructing a list you can create a regexp but the conditions here are irregular enough that the regex would still be quite large. That said, I worked on it a little more and I have an alternative regex-based approach that matches more than we need but still a finite number of values so maybe it's OK. See #4055.

Reply:
I updated the PR, there were way too many properties listed here. It's +43 bytes now.